src/components/Home/Main.jsx

```diff
@@ -1,378 +0,0 @@
-          />
-          <MainCard
-            imageUrl={
-              "https://images-cricketcom.imgix.net/news-1661351778510?auto=compress&dpr=2&w=1&h=200"
-            }
-            heading = {'Tilak, Umran and Kuldeep named in India A squad'} subHeading = {'India A will play three four-day games against New Zealand A in September'}
-          />
-          <MainCard
-            imageUrl={
-              "https://images-cricketcom.imgix.net/news-1661350106530?auto=compress&dpr=2&w=1&h=200"
-            }
-            heading = {'Gill set to play four County Championship games for Glamorgan'} subHeading = {'He will become only the third Indian player after Ravi Shastri and Sourav Ganguly to play for Glamorgan in the County Championship'}
-          />
-          <MainCard
-            imageUrl={
-              "https://images-cricketcom.imgix.net/news-1661186466215?auto=compress&dpr=2&w=1&h=200"
-            }
-            heading = {'Rahul disappoints, but Gill and ODIs are a match made in heaven'} subHeading = {'Sikandar Raza and Brad Evans tried their best but India managed to get over the line to complete a 3-0 clean sweep'}
-          />
-          <MainCard
-            imageUrl={
-              "https://images-cricketcom.imgix.net/news-1661394564658?auto=compress&dpr=2&w=1&h=200"
-            }
-            heading = {'Fit-again Matt Henry returns for Australia ODIs'}
-            subHeading = {'Meanwhile, Kyle Jamieson (back) and Adam Milne (achilles) were not considered due to injury'}
-          />
-        </Box>
-        <Box w="100%" h="260px" border="1px solid black" marginBottom="2rem">
-          <Box height="50px">
-            <Text
-              padding="16px"
-              fontSize="12px"
-              fontWeight="bold"
-              // borderBottom={"1px solid gray"}
-            >
-          DATA DIGEST
-
-            </Text>
-          </Box>
-          <Flex justifyContent={"space-evenly"} marginTop="1rem">
-            <Box height="192px" width="192px">
-              <Image
-                src={"https://s3.ap-south-1.amazonaws.com/image.cricket.com/datadigest/d2093c655fff970c893d62e6f355715b2ce27a26.png"}
-              />
-             
-            </Box>
-            <Box height="192px" width="192px">
-              <Image src="https://s3.ap-south-1.amazonaws.com/image.cricket.com/datadigest/453aaf421bc1087199fbd51833b77738cf4f6377.png" />
-              
-            </Box>
-            <Box height="192px" width="192px">
-              <Image src="https://s3.ap-south-1.amazonaws.com/image.cricket.com/datadigest/f62a6cf67cbce5037a4c02a023a5b806ef3534dd.png" />
-            
-            </Box>
-            <Box height="192px" width="192px">
-              <Image src="https://s3.ap-south-1.amazonaws.com/image.cricket.com/datadigest/4eec0f8505aa7b1172a8ca8126389c5b88384981.png" />
-            
-            </Box>
-          </Flex>
-        </Box>
-      </Box>
-
-    </Box>
-  );
```

Comment:
Your filesize shouldn't go beyond 100 or 200 lines. If it is going beyond this, then consider breaking it into different files